build/cmake/DefinePlatformSpecifc.cmake

```diff
@@ -58,8 +58,9 @@ if(MSVC)
     endif(WITH_MT)
 endif(MSVC)
 
-# GCC Specific
-if(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX)
+# GCC or Clang specific
+if(CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
```

Comment:
Unfortunately CMAKE_COMPILER_IS_CLANG doesn't exist (or at least didn't exist last time I researched the topic).